src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -78,21 +78,20 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
     >
       <PageTitle title={'New Project'} previousPages={[{ name: 'Projects', route: routes.PROJECTS }]} />
       <PageBlock title={''}>
-        <Grid container spacing={2}>
-          <Grid item xs={12} md={3}>
-            <FormControl>
-              <FormLabel>Change Request ID</FormLabel>
+        <Grid container sx={{ marginBottom: '24px' }}>
```

Comment:
I am nearly certain you do not want this marginBottom here, but I may be totally wrong.